src/Symfony/Bridge/Monolog/Logger.php

```diff
@@ -46,6 +46,16 @@ public function countErrors()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function clear()
```

Comment:
should be called "reset" for consistency

Reply:
Actually, I chose clear because that's how monolog calls similar functionality.

https://github.com/Seldaek/monolog/blob/fd8c787753b3a2ad11bc60c063cff1358a32a3b4/src/Monolog/Handler/TestHandler.php#L76